### DISCUSSION TITLE
Help me in setting the app in my local

### DISCUSSION THREAD

**Discussion by Prakash-Ravichandran** [supabase/supabase#17316] [Questions]
Hi @neiled @laktek @madx @RandalSchwartz @laran 

I had followed the [DEVELOPERS.md](https://github.com/supabase/supabase/blob/master/DEVELOPERS.md) & successfully installed the dependencies, while i am running the command  **"npm run dev"**  i am facing issues in running he app in my local please help me fix & i wanted to contributed to this project. Attached screenshot please help me, I am looking forward to contribute.

![image](https://github.com/supabase/supabase/assets/74542543/0d7d1304-da49-4b54-8a76-65b286db5f3a)

**Comment by MildTomato** [comment]
If your having issues running turbo, you can also cd into an app ie. `cd ./apps/www` and just `npm run dev`

**Reply by Prakash-Ravichandran** [reply]
1. hi @MildTomato @neiled @laktek @madx ,  i run ```cd ./apps/www``` and just ```npm run dev``` . But still i receive the below comments as ``` mising scripts ```, Can you guys pleas help me to run in local ??

-
![supabase_next_commit2](https://github.com/supabase/supabase/assets/74542543/634ca874-d7f5-4f02-b93d-26ec2a600f41)
